File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer>elements=new HashSet<>();
        for(int i=0;i<nums.length;i++){
            elements.add(nums[i]);
        }
        int maxSequence=0;
        for(int i:elements){
            if(!elements.contains(i-1)){
                int count=1;
                while(elements.contains(i+count)){
                    count++;
                }
                maxSequence=Math.max(maxSequence,count);
            }
           
        }
        return maxSequence;
        
    }
}

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        left=0
        right=k-1
        ans=[]
        while right<len(nums):
            ans.append(max(nums[left:right+1]))
            left+=1
            right+=1
        return ans
        